$NetBSD$

Add -L and -Wl,-R args to link to pick up Judy library.
Add "all" and "install" targets.
Remove / comment-out non-pkgsrc include and library paths..

--- Makefile.orig	2017-08-04 14:27:44.000000000 +0000
+++ Makefile
@@ -1,9 +1,11 @@
 # The following options seem to work fine on Linux, FreeBSD, and Darwin
 OPTIMIZE=-O2 -g
 CFLAGS=-Wall -Werror -pthread -fno-strict-aliasing
-INCPATH=-I/usr/local/include -I/opt/local/include -I/usr/local/ssl/include
+# INCPATH=-I/usr/local/include -I/opt/local/include -I/usr/local/ssl/include
 CC?=cc
 
+EXTRALPATH=-L${PREFIX}/lib -Wl,-R,${PREFIX}/lib
+
 # These additional options work on Solaris/gcc to which I have an access
 # (when combined with the options above, and CC=gcc).
 #EXTRALPATH=-L/usr/local/ssl/lib -Wl,-R,/usr/local/ssl/lib
@@ -18,6 +20,14 @@ CC?=cc
 #EXTRALIBS-lrt -lnsl
 #EXTRALINKING=-mt -lpthread
 
+all: validns
+
+install:
+	mkdir -p ${DESTDIR}/${PREFIX}/bin
+	install -c -m 555 validns ${DESTDIR}/${PREFIX}/bin
+	mkdir -p ${DESTDIR}/${PREFIX}/man/man1
+	install -c -m 444 validns.1 ${DESTDIR}/${PREFIX}/man/man1
+
 validns: main.o carp.o mempool.o textparse.o base64.o base32hex.o \
 	rr.o soa.o a.o cname.o mx.o ns.o \
 	rrsig.o nsec.o dnskey.o txt.o aaaa.o \
@@ -39,7 +49,7 @@ validns: main.o carp.o mempool.o textpar
 	    ipseckey.o cbtree.o mb.o mg.o mr.o minfo.o \
 	    afsdb.o x25.o isdn.o rt.o px.o kx.o \
 	    dlv.o dhcid.o nsap.o caa.o \
-	    -L/usr/local/lib -L/opt/local/lib $(EXTRALPATH) \
+	    $(EXTRALPATH) \
 	    -lJudy -lcrypto $(EXTRALIBS) $(EXTRALINKING)
 
 clean:
